src/dashboard/PointOfCareDashboard.jsx

```diff
@@ -145,15 +157,59 @@ export default class PointOfCareDashboard extends Component {
         return this.state.showPOC ? this._unselectedTdpIcon() : this._selectedTdpIcon();
     }
 
+    renderNextPatient = () => {
+        const dataAccess = new DataAccess('HardCodedMcodeV01DataSource');
```

Comment:
[JA1] Should the DataAccess we use here be 'HardcodedTabletMcodeV01DataSource'?

Reply:
Theoretically yes, but we actually should not be constructing a new DataAccess instance here. The dashboard takes dataAccess as a prop. Instead of constructing a new one, use `this.props.dataAccess`

Reply:
JA1 Resolved!